src/DynamoCoreWpf/Interfaces/IVisualizationManager.cs

```diff
@@ -68,6 +68,20 @@ public interface IVisualizationManager : IDisposable
         IRenderPackage CreateRenderPackageFromGraphicItem(IGraphicItem graphicItem);
 
         /// <summary>
+        /// An event triggered on the Selection of model.
+        /// </summary>
+        event Action<IEnumerable> RenderSelection;
+
+        /// <summary>
+        /// An event triggered on the Deletion of model.
+        /// </summary>
+        event Action<NodeModel> UpdateGeometryOnNodeDeletion;
```

Comment:
@ramramps Similar to above, can we rename this to `VisualizationManager.DeletionHandled`.